lenskit-core/src/main/java/org/grouplens/lenskit/data/dao/packed/BinaryRatingPacker.java

```diff
@@ -252,7 +251,6 @@ private void upgradeRatings(BinaryFormat newFormat) throws IOException {
 
         ByteBuffer oldBuffer = ByteBuffer.allocateDirect(format.getRatingSize());
         ByteBuffer newBuffer = ByteBuffer.allocateDirect(newFormat.getRatingSize());
-        MutableRating scratch = new MutableRating();
```

Comment:
Its worth pointing out. I find this code a lot easier to read when its doing "normal java stuff" instead of the mutable optimizations.